linera-service/src/node_service.rs

```diff
@@ -991,6 +991,9 @@ where
         let application_handler =
             axum::routing::get(util::graphiql).post(Self::application_handler);
 
+        let span = tracing::info_span!("node_service", port);
+        let _entered = span.enter();
```

Comment:
Shouldn't we avoid calling `enter` inside `async` functions? I think the general practice is to enter and leave on every poll: https://docs.rs/tracing/latest/tracing/span/index.html#closing-spans

Reply:
Good catch, thank you!